File: Solutions/2486. Append Characters to String to Make Subsequence/2486.java
class Solution {
    public int appendCharacters(String s, String t) {
        int i = 0, j = 0;
        int sLen = s.length(), tLen = t.length();
        char[] sc = s.toCharArray(), tc = t.toCharArray();
        while(i < sLen && j < tLen){
            if(sc[i] == tc[j]){
                j++;
            }
            i++;
        }
        return tLen - j;
    }
}

File: Solutions/2486. Append Characters to String to Make Subsequence/2486.py
class Solution:
    def appendCharacters(self, s: str, t: str) -> int:
        i, j = 0, 0
        sLen, tLen = len(s), len(t)
        while i < sLen and j < tLen:
            if s[i] == t[j]:
                j += 1
            i += 1
        return tLen - j